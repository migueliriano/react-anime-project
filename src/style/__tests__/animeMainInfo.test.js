import React from 'react';

import {
  MainInfoContainer,
  PosterContainer,
  PosterImage,
  AnimeDetails,
  Row,
  TitleDetail,
  Detail,
} from '../animeMainInfo';

describe('animeMainInfo style components', () => {
  it('Should render a MainInfoContainer Component', () => {
    const tree = renderer.create(<MainInfoContainer />).toJSON();
    expect(tree).toMatchSnapshot();
  });

  it('Should render a PosterContainer Component', () => {
    const tree = renderer.create(<PosterContainer />).toJSON();
    expect(tree).toMatchSnapshot();
  });

  it('Should render a PosterImage Component', () => {
    const tree = renderer.create(<PosterImage />).toJSON();
    expect(tree).toMatchSnapshot();
  });

  it('Should render a AnimeDetails Component', () => {
    const tree = renderer.create(<AnimeDetails />).toJSON();
    expect(tree).toMatchSnapshot();
  });

  it('Should render a Row Component', () => {
    const tree = renderer.create(<Row />).toJSON();
    expect(tree).toMatchSnapshot();
  });

  it('Should render a TitleDetail Component', () => {
    const tree = renderer.create(<TitleDetail />).toJSON();
    expect(tree).toMatchSnapshot();
  });

  it('Should render a Detail Component', () => {
    const tree = renderer.create(<Detail />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
