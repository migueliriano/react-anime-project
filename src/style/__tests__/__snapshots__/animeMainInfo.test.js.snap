// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`animeMainInfo style components Should render a AnimeDetails Component 1`] = `
.c0 {
  background-color: #f5f8f9;
}

<div
  className="c0"
/>
`;

exports[`animeMainInfo style components Should render a Detail Component 1`] = `
<span
  className=""
/>
`;

exports[`animeMainInfo style components Should render a MainInfoContainer Component 1`] = `
.c0 {
  border: solid 1px #edeeee;
  max-width: 300px;
  display: inline-block;
}

<div
  className="c0"
/>
`;

exports[`animeMainInfo style components Should render a PosterContainer Component 1`] = `
.c0 {
  padding: 15px;
}

<div
  className="c0"
/>
`;

exports[`animeMainInfo style components Should render a PosterImage Component 1`] = `
.c0 {
  max-width: 220px;
}

<img
  className="c0"
/>
`;

exports[`animeMainInfo style components Should render a Row Component 1`] = `
.c0 {
  padding: 10px 15PX;
  border-bottom: 1px solid #eee;
}

<div
  className="c0"
/>
`;

exports[`animeMainInfo style components Should render a TitleDetail Component 1`] = `
.c0 {
  width: 40%;
  display: inline-block;
  color: #0591da;
  text-transform: capitalize;
  font-weight: bold;
}

<div
  className="c0"
/>
`;
